### PR TITLE
Add new ebook landing page

### PR DESCRIPTION
**Pull request by aileenrose**
1st draft of ebook landing page.  This is the organic page.

### CODE REVIEW

**Comment by aileenrose** [comment]
Note this is the Organic landing page, not for use by PPC / Google ads.

**Comment by sd9** [comment]
Having the heading in the jumbotron would make the page more visually appealing. 

Also, it looks like the SVG ebook image is empty.

**Comment by ryan-blunden** [comment]
Having the heading in the jumbotron would make the page more visually appealing.

Also, it looks like the SVG ebook image is empty.

**Comment by aileenrose** [comment]
> Having the heading in the jumbotron would make the page more visually appealing.

Re:  we can A/B test this

> Also, it looks like the SVG ebook image is empty.

I switched image to jpg.

File: website/src/pages/resources/universal-code-search-ebook.tsx [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+                    </div>              
+                    <div className="row flex-wrap-reverse">
+                        <div className="col-md-4">
+                            <p>By Quinn Slack and Beyang Liu</p>
+                            <p>Highly productive engineering organizations make searching across massive codebases, comprehending unfamiliar code, and sharing institutional first-order concerns. To achieve this, engineering teams require a
+                            tool that provides universal code search.</p>
+                            
+                            <p>With this ebook, you will learn:</p>
+                            <ul>
+                                <li>what is universal code search</li>
```
Comment (vanesa) [inline website/src/pages/resources/universal-code-search-ebook.tsx]: As far as I know, if the list item is a full sentence, you need to capitalize the first letter. I would say all of the items would pass as full sentences.

**Comment by vanesa** [comment]
Agree with Ryan on the jumbotron proposition. A/B testing would be interesting.

File: website/src/pages/resources/universal-code-search-ebook.tsx (hunk 1 above)
Comment (aileenrose) [reply website/src/pages/resources/universal-code-search-ebook.tsx]: fixed

**Pull request merged**